lambdas/postgres-migration-async-operation/variables.tf

```diff
@@ -0,0 +1,65 @@
+
+variable "buckets" {
+  type    = map(object({ name = string, type = string }))
+  default = {}
+}
+
+variable "dynamo_tables" {
+  description = "A map of objects with the `arn` and `name` of every DynamoDB table for your Cumulus deployment."
+  type        = map(object({ name = string, arn = string }))
+}
+
+variable "prefix" {
+  type = string
+}
+
+variable "system_bucket" {
+  description = "The name of the S3 bucket to be used for staging deployment files"
```

Comment:
Not to fix in this PR -- this is consistent across all our modules, at least in the ones that have this variable, but I can't help but wonder if it's a useful description.   Up for debate.